samples/samples-powershell/.funcignore

```diff
@@ -0,0 +1,6 @@
+*.js.map
```

Comment:
Clean up this file to only include the necessary stuff, js and ts for example definitely don't need to be in here